OpenRA.Mods.Common/Scripting/Global/EffectGlobal.cs

```diff
@@ -32,5 +35,17 @@ public void Flash(string type = null, int ticks = -1)
 				if (fpe.Info.Type == type)
 					fpe.Enable(ticks);
 		}
+
+		[Desc("Dynamically adjusts the `GlobalLightingPaletteEffect` trait.")]
+		public void ChangeLighting(double red, double green, double blue, double ambient)
```

Comment:
Could we make this accept an HSLColor (which is what we use for the other color-using methods), and then add something to the HSLColor table to create a color from the specified RGB?